flake.nix

```diff
@@ -110,11 +110,30 @@
 
       # Standard flake attribute allowing you to add the villas packages to your nixpkgs
       overlays = {
-        default = final: prev: packagesWith final;
+        default = final: prev: (packagesWith final) // {
+          libiec61850 = prev.libiec61850.overrideAttrs {
```

Comment:
It would be nice if we could move this overlay into separate `.nix` file somewhere under `packaging/nix`. Ideally with a short comment on what it does.

Reply:
Thanks. I think the current state is good enough :)